test/testcafe/spec/ChallengeAuthenticatorOktaVerifyPushOnly_spec.js

```diff
@@ -0,0 +1,88 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+
+import ChallengeOktaVerifyPushPageObject from '../framework/page-objects/ChallengeOktaVerifyPushPageObject';
+import ChallengeFactorPageObject from '../framework/page-objects/ChallengeFactorPageObject';
+
+import xhrSelectAuthenticatorOVWithMethodSendPushOnly from '../../../playground/mocks/data/idp/idx/authenticator-verification-okta-verify-send-push-autoChallenge';
+import xhrAuthenticatorOVPushWithAutoChallenge from '../../../playground/mocks/data/idp/idx/authenticator-verification-okta-verify-push-autoChallenge';
+
+const requestLogger = RequestLogger(
+  /introspect|\/challenge/,
+  {
+    logRequestBody: true,
+    stringifyRequestBody: true,
+  }
+);
+
+const mockOktaVerifySendPushOnly = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrSelectAuthenticatorOVWithMethodSendPushOnly);
+
+const mockChallengeOVSendPush = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrSelectAuthenticatorOVWithMethodSendPushOnly)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge')
+  .respond(xhrAuthenticatorOVPushWithAutoChallenge);
+
+fixture('Challenge Authenticator Okta Verify Push Only With Auto Challenge Form');
+
+async function setup(t) {
+  const challengeOktaVerifyPushPageObject = new ChallengeOktaVerifyPushPageObject(t);
+  await challengeOktaVerifyPushPageObject.navigateToPage();
+  return challengeOktaVerifyPushPageObject;
+}
+
+test.requestHooks(mockOktaVerifySendPushOnly)('should load view with a button and a checkbox when push is the only method type', async t => {
+  const AUTO_CHALLENGE_CHECKBOX_SELECTOR = '[name="authenticator.autoChallenge"]';
+  const AUTO_CHALLENGE_CHECKBOX_LABEL_SELECTOR = '[data-se-for-name="authenticator.autoChallenge"]';
```

Comment:
we should move these 2 constants to the page object, that way the page object "knows" how to find or click autoChallenge checkbox, and then we don't need to define them in all the tests specs

Reply:
the value of these constants differs based on the view. And test spec of both the views share the same page object. So the page object has function that takes this selector as arguments. to prevent from creating 4 functions I kept the constants in the test file itself. does it make sense?

Reply:
oh... can we use another selector? I see you are using `[data-se="o-form-fieldset-autoChallenge"]` in the View, would that work?

Reply:
even they differ. Its automatically set based on the remediation info. And that info differs for these views. 
For `ChallengeAuthenticatorDataOktaVerifyPushOnlyBody` checkbox container has attribute as `[data-se="o-form-fieldset-authenticator.autoChallenge"]`

For `ChallengePushView` checkbox container has attribute as `[data-se="o-form-fieldset-autoChallenge"]`